src/vcpkg/base/files.cpp

```diff
@@ -1556,7 +1566,24 @@ namespace vcpkg
             {
                 while (b != e)
                 {
-                    ret.push_back(from_stdfs_path(b->path()));
+#if VCPKG_USE_STD_FILESYSTEM
+                    if (b->is_directory(ec))
+#else  // ^^^ VCPKG_USE_STD_FILESYSTEM // !VCPKG_USE_STD_FILESYSTEM vvv
```

Comment:
Didn't we say we wanted to remove these?

Reply:
Not as far as I know. Compare with 1666

Reply:
I strongly believe we should stop adding them because they (in certain circumstances) trigger bugs in clang-format between different OS's.

Reply:
I believe it doesn't trigger differing behavior in clang-format 12